ambry-store/src/test/java/com.github.ambry.store/IndexTest.java

```diff
@@ -49,6 +49,7 @@
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.concurrent.atomic.AtomicReference;
+import junit.framework.Assert;
```

Comment:
this looks like bad import. We never use `junit.framework.Assert`. We use `org.junit.Assert` (which is already available in this class).